src/scss/layout/_footer.scss

```diff
@@ -48,3 +52,9 @@ $two-column: 45em;
 
   font-size: var(--xsmall);
 }
+
+.oddnews-signup {
+  background-color: var(--clip-bg, var(--callout));
+  grid-area: oddnews;
+  padding: 0 var(--gutter) var(--gutter);
```

Comment:
@sanajaved7 I removed the margin-top property on the h3 in this div and instead edited the padding so we don't have to override the global style.

Reply:
Thank you @stacyk! 🙏🏼